Frontend/CO.CDP.OrganisationApp/Session.cs

```diff
@@ -8,7 +8,7 @@ public class Session(IHttpContextAccessor httpContextAccessor) : ISession
     public const string RegistrationDetailsKey = "RegistrationDetails";
     public const string ConnectedPersonKey = "ConnectedPerson";
 
-    public T? Get<T>(string key)
+    public virtual T? Get<T>(string key)
```

Comment:
I added it because I needed to override that Get method at one point, but I'm not doing that any more so this can be reverted.
I'll change it back...